docs/javascripts/downloads.js

```diff
@@ -0,0 +1,191 @@
+(function() {
+  const downloadsTableWrapper = document.getElementById('downloads-table-wrapper');
+
+  if (downloadsTableWrapper) {
+    const appId = '52e3297d-ceeb-45c9-bbe4-6069fe920760';
+    const services = [
```

Comment:
Instead of having to build the full table here in JavaScript, what about keeping it in the markdown file (wrapped in the `<div>` you have), and just look up the last column of each row to plug in the generated values down here to those cells ⬇️ ? Would be less JavaScript needed I'm guessing :)

Reply:
Should we rely on the indexes in the table?
If we change the markdown document the js-file info will end up in incorrect cells.

Reply:
Just assume the last `td` in each row is reserved for your content, e.g. `table.querySelectorAll('tr td:last-child')`